rllib/algorithms/dreamerv3/utils/summaries.py

```diff
@@ -115,7 +116,7 @@ def report_dreamed_trajectory(
 
 def report_predicted_vs_sampled_obs(
     *,
-    results,
+    metrics,
```

Comment:
We pass in the main Algo's metrics object now and just log into that (instead of returning altered dicts).